File: MAHAds/mah-ads/src/main/java/com/mobapphome/mahads/mahfragments/MAHFragment.java
package com.mobapphome.mahads.mahfragments;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentActivity;


public class MAHFragment extends Fragment {


    public FragmentActivity getActivityMAH() throws MAHFragmentExeption {
        if (getActivity() != null) {
            return getActivity();
        } else {
            throw new MAHFragmentExeption("MAHFragment getActivity() method returns null");
        }
    }

}

File: MAHAds/mah-ads/src/main/java/com/mobapphome/mahads/mahfragments/MAHFragment.kt
package com.mobapphome.mahads.mahfragments

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentActivity


class MAHFragment : Fragment() {


    val activityMAH: FragmentActivity
        @Throws(MAHFragmentExeption::class)
        get() {
            if (activity != null) {
                return activity
            } else {
                throw MAHFragmentExeption("MAHFragment getActivity() method returns null")
            }
        }

}
